theforeman.org/yaml/jobs/packaging_build_deb_dependency.yaml

```
@@ -29,7 +29,7 @@
           values:
           - debian
     execution-strategy:
-      combination-filter: '(onlyos == "all" || onlyos == os) && !((version == "2.4" || version == "2.3") && os == "focal")'
+      combination-filter: '(onlyos == "all" || onlyos == os) && !((version == "2.5" || version == "2.4" || version == "2.3") && os == "focal")'
```

no, focal is supported on 2.5 :)

```suggestion
      combination-filter: '(onlyos == "all" || onlyos == os) && !((version == "2.4" || version == "2.3") && os == "focal")'
```